.github/workflows/deploy.yml

```
@@ -81,11 +81,20 @@ jobs:
       - name: Install dependencies
         run: ${{ steps.detect-package-manager.outputs.manager }} ${{ steps.detect-package-manager.outputs.command }}
 
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
```

Can we change the date format to: 

```suggestion
        run: echo "::set-output name=date::$(date +'%m/%d/%Y')"
```